frontend/packages/db/supabase/migrations/20250423123348_invite_organization_member.sql

```diff
@@ -0,0 +1,2 @@
+-- This empty .sql file is required because a migration with this filename was already applied in production.
+-- It will no longer be needed if the production database is ever reset.
```

Comment:
I see, I want to squash migrations before first release!

Reply:
I agree with squashing. I think it's fine to squash all the files, not just the empty ones! :)